docs/multi-storage-transactions.md

```diff
@@ -1,56 +1,60 @@
-# Multi-storage Transactions
+# How to Configure ScalarDB for Multi-Storage Transactions
```

Comment:
This page describes how multi-storage transactions work, in addition to information on how to configure it. So, the original title might be better since the new title seems only to focus on the configuration?

Reply:
Sure, I'm OK with reverting to the original title. I did the same for the `two-phase-commit-transactions.md` doc in this PR too since I changed the title in that doc to follow the same format.